docs/05-technical-reference/00-architecture/evnt-01-architecture.md

```diff
@@ -34,7 +34,7 @@ Eventing Controller manages the internal infrastructure in order to receive an e
 
 ## JetStream
 
-Kyma now supports JetStream, which is a persistence offering from NATS, that guarantees `at least once` delivery. It is built-in within our default NATS backend. Follow the [tutorial](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md) to enable Kyma with JetStream.
+Kyma now supports JetStream by default, which is a persistence offering from NATS, that guarantees `at least once` delivery. It is built-in within our default NATS backend.
```

Comment:
(nit)
I am not sure it's the best to formulate this. Perhaps something like: `JetStream NATS is enabled in Kyma by default, which is...`